internal/compiler/widgets/material-base/button.slint

```diff
@@ -8,17 +8,19 @@ import { Typography, Palette, Elevation } from "styling.slint";
 export component Button {
     callback clicked;
 
-    in property<string> text <=> i-text.text;
+    in property<string> text;
     in property<bool> enabled <=> i-state-layer.enabled;
     in property<bool> checkable;
     in property<image> icon;
     in property <bool> primary;
     out property<bool> has-focus: i-state-layer.has-focus;
     out property<bool> pressed: self.enabled &&  i-state-layer.pressed;
     in-out property<bool> checked;
+    property <brush> text-color: root.primary ? Palette.on-primary : Palette.on-secondary-container;
```

Comment:
Please add `text-color` and `text-opacity` property to the start of the component and add a `private`, check `listview.slint`. Order should be 1. private properties, 2. callbacks, 3. in properties, 4. out properties, 5. in-out properties, 6. children, 7. functions, 8. states, 9. animations.

Reply:
Done